src/Hit/Git.hs

```diff
@@ -58,6 +59,13 @@ runCommit (T.strip -> msg)
 runPush :: IO ()
 runPush = getCurrentBranch >>= \branch -> "git" ["push", "-u", "origin", branch]
 
+-- | @hit resolve@ command.
+runResolve :: IO ()
+runResolve = do
+    branch <- getCurrentBranch
+    runHop $ Just "master"
+    when (branch /= "master") $ "git" ["branch", "-D", branch]
```

Comment:
I think you can also pass `Maybe Text` as a branch name here and use `master` by default, otherwise -- given name. So you can write `hit resolve develop` or something like that